packages/jam/jam-host-calls/externalities/partial-state.ts

```diff
@@ -151,6 +151,8 @@ export enum NewServiceError {
   InsufficientFunds = 0,
   /** Service is not privileged to set gratis storage. */
   UnprivilegedService = 1,
+  /** Trying to create a service with an ID that already exists. */
+  ServiceAlreadyExists = 2,
```

Comment:
```suggestion
  /** Registrar attempting to create a service with already existing ID. */
  RegistrarServiceIdAlreadyTaken = 2,
```

Note it's very different from creating duplicated services, which fails the entire accumulation and we can hit the second case even with registrar.